lib/msal-browser/src/interaction_client/StandardInteractionClient.ts

```diff
@@ -93,6 +116,26 @@ export abstract class StandardInteractionClient extends BaseInteractionClient {
         return validLogoutRequest;
     }
 
+    /**
+     * Parses preferred_username ID Token Claim out of AccountInfo object to be used as 
+     * logout_hint in end session request.
+     * @param account 
+     */
+    protected getLogoutHintFromIdTokenClaims(account: AccountInfo): string | null {
+        const idTokenClaims: IdTokenClaims | undefined = account.idTokenClaims;
+        if (idTokenClaims) {
+            if (idTokenClaims.preferred_username) {
```

Comment:
We should be using the `login_hint` claim, not preferred_username, right?

Reply:
(this is a new optional id token claim that must be enabled by the app first)

Reply:
Right, fixed, thanks!